shared/gh/partials/admin-module-item.html

```diff
@@ -29,7 +29,9 @@
             <ul class="list-group hide">
                 <%= _.partial('admin-module-item', {
                     'data': d.Series,
-                    'isChild': true
+                    'isChild': true,
+                    'state': state,
+                    'moduleMatch': parseInt(state.module, 10) === d.id
```

Comment:
If a series is borrowed into different modules and the user clicks the first one of them, we need to make sure that only the clicked one is marked as 'active' and not the same series in other modules as well. This is where the `moduleMatch` comes in to play. If the clicked series ID matches the module its in, we're going to highlight it as active.